web/src/features/Dashboard/components/DashboardVersionCard.tsx

```diff
@@ -202,12 +209,23 @@ const DashboardVersionCard = (props: Props) => {
     }
   }, [history.location.search]);
 
+  useEffect(() => {
+    if (latestDeployableVersionErrMsg instanceof Error) {
+      setState({
+        latestDeployableVersionErrMsg: `Failed to get latest deployable version: ${latestDeployableVersionErrMsg.message}`,
+      });
+      return;
+    }
+
+    setState({
+      latestDeployableVersionErrMsg: "Something went wrong, please try again.",
+    });
+  }, [latestDeployableVersionErrMsg]);
```

Comment:
This is just pulling the error handling that was in the fetch handler that we replaced with the ook